client/css/pagination.scss

```diff
@@ -0,0 +1,38 @@
+.paginate-nav {
```

Comment:
@dwoznicka could you consider moving colors and paddings values to separate variable ?